help/search/_posts/2015-02-09-search-syntax.md

```diff
@@ -61,6 +61,21 @@ aff:(China OR "Hong Kong" Taiwan) | aff:((China OR "Hong Kong") AND Taiwan)
 aff:(China OR "Hong Kong" NOT Taiwan) |aff:(China OR ("Hong Kong" NOT Taiwan))
 aff:(China OR "Hong Kong" -Taiwan) | aff:((China OR "Hong Kong") NOT Taiwan)
 
+For a more heavy handed exploration of the search syntax, feel free to visit [search parser details](help/gory_details/_posts/2021-05-23-search-parser.md).
+
+
+### Wildcards, Proximity, and Regular expression search
+
+Example Query                        | Explanation
+-------------------------------------------------|------------------------------------------------
+author:&ldquo;huchra, jo*"                | multi-character wildcard; search for papers written by `huchra, john`, `huchra, jonathan`, `huchra, jolie`, and anything in between
+author:&ldquo;bol?,"               | single-character wildcard; in this case we may get back `bolt`, `boln`, `bolm`
+title:(map NEAR5 planar)                          | instead of a phrase search, we can ask the search engine to consider words be close to each other -- the maximum allowed distance is 5; the operator must be written as `NEAR[number]`; in this example the search terms can appear in any order but there will be at most 5 other terms between (not counting stopwords such as `a`, `the`, `in`...). **The proximity search must be used only against fielded search, i.e. inside one index. You cannot use proximity search to find an author name next to a title**
+grant:/nag[1-9]+/                    | Regular expression searches are possible but are less useful than you might expect. Firstly, the regex can match only against indexed tokens - i.e. it is not possible to search for multiple words next to each other. So in practice, this type of search is more useful for fields that contain `string` tokens (as opposed to `text` tokens). In practice, this means that a field which contains many words (such as `title`, `abstract`, `body`) is a text field, but a field with only limited number of values is typically defined as a `string` - for example, `author`, `grant`, `simbid`. You can use regex in both `string` and `text` fields but you have to be aware that regular expression is only going to match **indexed tokens**. Do not think that you can search for several token using one regex expression. A little bit or more of experimentation (test queries) should be enough to help you determine your 'adversary'. For description of allowed regex patterns, please see: [Lucene documentation](https://lucene.apache.org/core/7_0_1/core/org/apache/lucene/util/automaton/RegExp.html)
```

Comment:
I don't think we should list. The wildcard and other searches work equally in both types (and other subtypes we have). I've just tested reader (which is a `string`) and we know it works in `text` fields - or have you seen wildcards not working in some?

as for `grant` - currently, we have 43073 - I can't say if that is too little but if the users beging yelling about it, it will spur us to extract more -- and that's a good thing. I don't cherish the idea of people trying regex for all first names in the big index that authors are (because they are going to do exactly that).

Reply:
simbid is really a bad example, I don't expect anybody to use it (it's just a sequential identifier we index in order to search for object names).  Grant still exists, but it's not maintained and we should probably change it to `award` or something along those lines.  Aren't there better string examples?  Keyword maybe?

Also, I would add an additional clarifying language, in there:

You can use regex in both `string` and `text` fields but you have to be aware that regular expression is only going to match **indexed tokens**.  In the case of text fields (such as title, abstract, or body), indexed tokens are individual words.  In the case of string fields tokens may be multi-word combinations, depending on the specific field.  For example, a token for the author field is `huchra, john`.  In all cases the tokens are normalized (typically by lowercasing the input data).